src/components/dashboard/charts/simplified.js

```diff
@@ -127,6 +127,12 @@ export const SimpleStatistics = ({
 						display: true,
 						labelString: t(xLabel || 'TIMEFRAME'),
 					},
+					ticks: {
+						autoSkip: false,
+						callback: function(tick, index, array) {
```

Comment:
This fixes the last data points that are missing 
![missing datapoint](https://user-images.githubusercontent.com/22402693/75465622-8ed9f480-5991-11ea-811f-e74fe791cd9c.png)